src/project-tests/tribute-page-tests.js

```diff
@@ -42,41 +59,85 @@ export default function createTributePageTests() {
         );
       });
 
-      it(`I should see a <div> element with corresponding
-      id="img-div".`,
-      function() {
-        assert.isNotNull(document.getElementById('img-div'));
-      });
-
-      it(`Within the "img-div" element, I should see an <img> element
-      with a corresponding id="image".`,
-      function() {
-        assert.isNotNull(document.getElementById('image'));
-        assert.strictEqual(
-          $('#img-div').find('#image').length,
-          1,
-          'Element is not a child of id="img-div" '
-        );
-      });
-
-      it(`Within the "img-div" element, I should see an element with
-      a corresponding id="img-caption" that contains textual content describing
-      the image shown in "img-div".`,
-      function() {
-        assert.isNotNull(document.getElementById('img-caption'));
-        assert.strictEqual(
-          $('#img-div').find('#img-caption').length,
-          1,
-          'Element is not a child of id="img-div" '
-        );
-        const captionContents =
-          document.getElementById('img-caption').innerText;
-        assert.isAbove(
-          captionContents.length,
-          0,
-          'Element does not have any content '
-        );
-      });
+      // Tests for old projects when id=main is a div element
+      if (mainElement === 'div') {
+
+        it(`I should see a <div> element with corresponding
```

Comment:
By simply changing the phrase to `I should see either a <figure> or <div> element with corresponding...`, keeping the `id="img-div"` for both cases, we only need to change the wording of the user story. Is that correct?

Reply:
Yes, that is correct. Are we sure that id name won't be confusing on a figure element?

I have all the changes needed ready.

1. Update the project to use the old ids (for both the div/figure and div/figcaption elements).
2. Revert the test back to the old state, with just the extra line "either a `<figure>` or `<div>`".

@tbushman Should I just push the changes?

Reply:
@lasjorg Yes. Sorry to confuse matters. It will be good to keep the code simpler since the tests only test the `id` and not the `tag` , which I hadn't realized early in the discussion.